src/pages/storage/CreateStoragePool.tsx

```diff
@@ -8,21 +8,23 @@ import SubmitButton from "components/SubmitButton";
 import { useFormik } from "formik";
 import * as Yup from "yup";
 import { useNavigate, useParams } from "react-router-dom";
-import { LxdStoragePool } from "types/storage";
 import { queryKeys } from "util/queryKeys";
-import { zfsDriver, btrfsDriver } from "util/storageOptions";
+import { zfsDriver } from "util/storageOptions";
 import { testDuplicateStoragePoolName } from "util/storagePool";
 import StoragePoolForm, {
   StoragePoolFormValues,
+  storagePoolFormToPayload,
 } from "./forms/StoragePoolForm";
 import { useClusterMembers } from "context/useClusterMembers";
 import FormFooterLayout from "components/forms/FormFooterLayout";
+import { MAIN_CONFIGURATION } from "./forms/StoragePoolFormMenu";
 
 const CreateStoragePool: FC = () => {
   const navigate = useNavigate();
   const notify = useNotify();
   const queryClient = useQueryClient();
   const { project } = useParams<{ project: string }>();
+  const [section, setSection] = useState(MAIN_CONFIGURATION);
```

Comment:
Probably better to move this state into `StoragePoolForm` as it is not needed here.

Reply:
good point! Thanks